docs/reference/integrations/snowflake.rst

```diff
@@ -60,3 +60,8 @@ Additional Notes
     which means that you will have to rerun ``great_expectations init`` after these packages are installed.  Except for
     having to switch back and forth between ``great_expectations init`` and your Terminal ``shell``, the procedure will
     continue without any problems, because Great Expectations will pick up from where it left off.
+#.
+    When using the Snowflake dialect, `SqlAlchemyDataset` will create a **transient** table instead of a **temporary**
```

Comment:
Can you please add a code snippet to illustrate the new point about transient queries?

Reply:
@abegong I reviewed your last change request in terms of the code snippet.  I am not sure what code snippet can be illustrated here, since the transient tables are brought to the user's attention during the CLI experience.  So the user simply needs to choose (1, 2, 3, etc.) when there are multiple transient tables available (or the suggested default, when there is only one).  So I am wondering if there is anything else to do here.  Thanks. /cc @jcampbell